openapi/paths/tags-configurations.yaml

```
@@ -0,0 +1,54 @@
+get:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Retrieve tags configuration list
+  operationId: GetTagsConfigurationCollection
+  x-sdk-operation-name: getAll
+  description: Retrieve a list of tags configuration.
+  responses:
+    '200':
+      description: List of tags configuration retrieved.
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: ../components/schemas/TagUntag/TagUntagRule.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+post:
+  x-products:
+    - Core
+  tags:
+    - Tags
+  summary: Create tags configuration
+  operationId: PostTagConfiguration
+  x-sdk-operation-name: create
+  description: Create tags configuration.
```

issue: [based on our style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#operations)
```suggestion
  description: Creates a tags configuration.
```